### PR TITLE
LPS-118144 Create radio option for publication revert screen

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/edit_ct_collection.jsp [hunk 1]
```diff
@@ -61,36 +61,83 @@ portletDisplay.setShowBackIcon(true);
...
+					<aui:validator name="maxLength"><%= ModelHintsUtil.getMaxLength(CTCollection.class.getName(), "description") %></aui:validator>
+				</aui:input>
+
+			<c:if test="<%= revert %>">
+					<aui:fieldset cssClass="change-lists-fieldset" label="when-do-you-want-to-publish" markupView="lexicon">
+						<div class="col-10 row">
+							<div class="col-5">
+								<div class="autofit-row">
+									<div class="autofit-col">
+										<input checked="<%= true %>" class="field" id="<portlet:namespace/>publishTimeNow" name="<portlet:namespace/>publishTime" type="radio" value="now" />
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/edit_ct_collection.jsp:95]: We want to force the user to decide, so neither option should be checked.

Comment (stsquared99) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/edit_ct_collection.jsp:95]: got it

Comment (Preston-Crary) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/edit_ct_collection.jsp:95]: Also make sure the submit button is grayed out until they choose an option.

**Comment by Preston-Crary** [comment]
Please see my comments and resend, thanks!

**Pull request closed without merging**